packages/google_maps_flutter/example/lib/map_ui.dart

```diff
@@ -31,6 +31,8 @@ class MapUiBody extends StatefulWidget {
 class MapUiBodyState extends State<MapUiBody> {
   MapUiBodyState();
 
+  GoogleMapController mapController;
```

Comment:
could you create a new page and add it to `main.dart`, i'd like to leave this one without having a controller. Also that way we can maybe get around to having a text field that gets populated with the lat lng bounds on button press rather than an alert dialog. What do you think?

Reply:
done